Playground/js/mainWebGPU.js

```diff
@@ -410,6 +411,30 @@ class Main {
         this.parent.settingsPG.setScriptLanguage();
         // Check if it's mobile mode. If true, switch to full canvas by default
         this.parent.menuPG.resizeBigCanvas();
+
+        // HotKeys
+        document.onkeydown = function (e) {

```

Comment:
No conflict with the code editor?

Reply:
Code editor does not use these combinations. Other editor's combinations work fine

Reply:
You rock! Thanks a ton!